schema.ts

```diff
@@ -167,6 +167,8 @@ export enum ValueHintType {
    * Indicates to render a boolean value as a toggle.
    */
   Toggle = 'toggle',
+  /** @hidden */
+  CodaInternalRichText = 'codaInternalRichText',
```

Comment:
I would like to see more code documentation here to remind us what this field is used for. 

If we were ever to expose this to external developers, I think we'd use a new hint type name anyway. 

cc @ekoleda-codaio for sdk changes & docs for sanity check.

Reply:
Sure - I can add some more comments, but leave `@hidden` in place.

Reply:
LGTM. It's hard to hide things completely, but the name here also reflects the limited availability.